zulip_bots/zulip_bots/bots/baremetrics/test_baremetrics.py

```diff
@@ -0,0 +1,37 @@
+from zulip_bots.test_lib import BotTestCase
+
+class TestBaremetricsBot(BotTestCase):
+    bot_name = "baremetrics"
+
+    def test_bot_responds_to_empty_message(self) -> None:
+        # Offline query.
+        with self.mock_config_info({'api_key': 'lk_bszeiVWvcScrPXbGR0S1A'}):
```

Comment:
It looks like you are using your actual API key here (you should never do that! :)). For all the tests in this file, you should use `mock_http_conversation()` with your fixtures to simulate the http traffic, so that the bot tests are not dependant on internet access. You can check out the giphy bot for an example on how to do this.